test/src/components/traces/traces.spec.jsx

```diff
@@ -260,7 +275,7 @@ function createServiceStubStore() {
 
 function createSearchableKeysStore(results) {
     const store = new SearchableKeysStore();
-    store.keys = results;
+    store.keys = observable.array(results);
```

Comment:
Made the keys returned by searchableKeysStore an observable array to match application prop types. Resolves prop type issue with the options prop of Autocomplete.